### ISSUE TITLE
Delimited text message template stuck on 'parsing...', won't create message template tree

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#1452]
Open source transformer
Set inbound message template data type to 'delimited text'
Paste in value 0071,2001
Status bar displays 'Parsing...' and activity bar keeps bouncing
Message tree empty

Imported Issue. Original Details:
Jira Issue Key: MIRTH-1453
Reporter: lmann
Created: 2010-06-20T19:22:25.000-0700

**Comment by rbeckman-nextgen** [comment]
Connect reproduce this in the latest code.  Is there any other way of reproducing this?  Did you reproduce this in 2.0.0 beta or 1.8.2?

Imported Comment. Original Details:
Author: jacobb
Created: 2010-06-24T11:12:08.000-0700

**Comment by rbeckman-nextgen** [comment]
This is only occurring in the 2.0.0 beta.  1.8.2 on same machine works fine.  As I type I'm now wondering if the startup script is using the system default JVM on both?  Not on that machine right now so I will review and update the issue later today with that detail.

Imported Comment. Original Details:
Author: lmann
Created: 2010-06-24T12:44:21.000-0700

**Comment by rbeckman-nextgen** [comment]
Any update on this issue?  Still cannot reproduce.

Imported Comment. Original Details:
Author: jacobb
Created: 2010-06-28T15:59:52.000-0700

**Comment by rbeckman-nextgen** [comment]
Ensured starting both versions on same JVM, same issue.  However I've downloaded beta 2 and the issue is now resolved.  

One possibly relevant observation was that beta 1 would not hold the connector data types when set on the summary tab.  They did reflect settings made on the source via the transformer editor.  I see that this is now working with beta 2.  I've set to delimited/HL7x2 and HL7x2 and  the source transformer is correctly setting inbound connector to delimited and outbound to HL7v2.

Since working in beta 2 feel free to close issue.

Imported Comment. Original Details:
Author: lmann
Created: 2010-07-06T19:50:32.000-0700
